src/app/elastic/utils.py

```diff
@@ -1,8 +1,6 @@
 from typing import Union
 
 from elasticsearch_dsl import connections
-from elasticsearch_dsl.response import AggResponse
-from glom import merge
```

Comment:
If glom isn't used anywhere else it can also be removed from the requirements/pyproject dependencies?

Reply:
Good point - I removed glom, a few other dependencies and more unused files :)